lib/mock_suey/type_checks/sorbet.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+gem "sorbet-runtime", "~> 0.5"
+require "sorbet-runtime"
+require "set"
+require "pathname"
+
+require_relative "../sorbet_rspec"
```

Comment:
We're using Ruby Next, so `require_relative` should be avoided for files using modern syntax. I guess, that's why tests are failing for Ruby <3.1.

```suggestion
require "mock_suey/sorbet_rspec"
```